dev/com.ibm.ws.springboot.support.web.server/src/com/ibm/ws/springboot/support/web/server/internal/WebInstance.java

```diff
@@ -229,4 +221,28 @@ public void stop() {
         }
     }
 
+    @Override
+    public boolean isEndpointConfigured() {
+        VirtualHost v = getVirtualHost("default_host");
```

Comment:
I don't think we should wait for the default virtual host here (with timeout of 30 seconds).  If the default_host VirtualHost service is not already available we should just return false.